src/java.base/share/classes/jdk/internal/math/DoubleToDecimal.java

```diff
@@ -35,7 +36,10 @@
 /**
  * This class exposes a method to render a {@code double} as a string.
  */
-public final class DoubleToDecimal {
+public final class DoubleToDecimal extends ToDecimal {
+    public static DoubleToDecimal LATIN1 = new DoubleToDecimal(ToDecimal.LATIN1);
+    public static DoubleToDecimal UTF16  = new DoubleToDecimal(ToDecimal.UTF16);
+
     /*
      * For full details about this code see the following references:
```

Comment:
We should copy these comments to `ToDecimal`; otherwise, it's not obvious what references are being made in comments there.